source/includes/documentation/_vue_js_elements.md

```diff
@@ -568,50 +568,119 @@ and [JDI vuetify page tests for expansion panels](https://github.com/jdi-testing
+    </div>
+  </div>
+</footer>
+```
+
+| Method                                      | Description                                                                | Return Type  |
+|---------------------------------------------|----------------------------------------------------------------------------|--------------|
+| **is()**                                    | Returns Assert class                                                       | FooterAssert |
+| **isPadless()**                             | Checks that element is padless                                             | boolean      |
+| **isAbsolute()**                            | Checks that element is absolute                                            | boolean      |
+| **isFixed()**                               | Checks that element is fixed                                               | boolean      |
+| **has()**                                   | Returns Assert class                                                       | FooterAssert |
+| **color()**                                 | Returns css attribute background-color as String Value                     | String       |
+| **waitFor()**                               | Returns object for work with assertions                                    | FooterAssert |
+| **shouldBe()**                              | Returns object for work with assertions                                    | FooterAssert |
+| **verify()**                                | Returns object for work with assertions                                    | FooterAssert |
+| **assertThat()**                            | Returns object for work with assertions                                    | FooterAssert |
+| **classes()**                               | Gets all element's classes as list                                         | List<String> |
+| **doubleClick()**                           | Double clicks on the element                                               | void         |
+| **dragAndDropTo(int x, int y)**             | Drags and drops element to certain coordinates                             | void         |
+| **dragAndDropTo(WebElement to)**            | Drags and drops element to another element                                 | void         |
+| **getLocation()**                           | Gets element location as point                                             | Point        |
+| **getSize()**                               | Gets element size                                                          | Dimension    |
+| **getTagName()**                            | Gets element tag name                                                      | String       |
+| **getText()**                               | Gets element text                                                          | String       |
+| **getValue()**                              | Gets element text                                                          | String       |
```

Comment:
both methods get 'text'? just to clarify :)

Reply:
Yes, 
.getText() {return core().getText()}  _(so, .core() returns UIElement, and then .getText() returns UIElement test)_
.getValue() {return getText()}
Not sure, maybe sometimes JDI cannot getText() from some objects and have to perform core().getText() to get access to object's text